From 22cbe086b255daa87e3a99105bc16fd9819d485d Mon Sep 17 00:00:00 2001
From: Matteo Bertozzi <matteo.bertozzi@cloudera.com>
Date: Mon, 23 Nov 2015 16:05:05 -0800
Subject: [PATCH 285/295] HBASE-14799 Commons-collections object
 deserialization remote command execution
 vulnerability (addendum)

Reason: Security
Author: Andrew Purtell
Ref: CDH-34565

Change-Id: I43c33c0464926f3aac3a6564a249e36cdb8fe9c1
---
 .../TestHbaseObjectWritableFor96Migration.java     |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestHbaseObjectWritableFor96Migration.java b/hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestHbaseObjectWritableFor96Migration.java
index 6541278..4e34ea6 100644
--- a/hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestHbaseObjectWritableFor96Migration.java
+++ b/hbase-server/src/test/java/org/apache/hadoop/hbase/security/access/TestHbaseObjectWritableFor96Migration.java
@@ -75,14 +75,17 @@ import org.apache.hadoop.hbase.filter.WhileMatchFilter;
 import org.apache.hadoop.hbase.regionserver.HRegion;
 import org.apache.hadoop.hbase.regionserver.RegionOpeningState;
 import org.apache.hadoop.hbase.regionserver.wal.HLogKey;
+import org.apache.hadoop.hbase.testclassification.MediumTests;
 import org.apache.hadoop.hbase.wal.WAL;
 import org.apache.hadoop.io.MapWritable;
 import org.apache.hadoop.io.Text;
 import org.apache.hadoop.io.Writable;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 
 import com.google.protobuf.Message;
 
+@Category(MediumTests.class)
 public class TestHbaseObjectWritableFor96Migration {
 
   @Test
-- 
1.7.9.5

